README.md

```diff
@@ -1,7 +1,13 @@
 # TypedStruct
 
 [![Build Status](https://travis-ci.com/ejpcmac/typed_struct.svg?branch=develop)](https://travis-ci.com/ejpcmac/typed_struct)
-[![hex.pm version](http://img.shields.io/hexpm/v/typed_struct.svg?style=flat)](https://hex.pm/packages/typed_struct)
+[![Module Version](https://img.shields.io/hexpm/v/typed_struct.svg)](https://hex.pm/packages/typed_struct)
```

Comment:
Why did you replace `hex.pm` by `Module`? A module does not have a version by itself\*, or at least not the package version. Also, precising `hex.pm` let the user know it is the version hosted on `hex.pm`, that can temporarily differ from the latest tag on GitHub.

Secondly, why did you remove `style=flat`? I don’t know what is the default, but anyway explicit style is always better.

\* It could indeed have a version if we use hot reloading, but this is out of scope here.